frontend/src/app/modules/interventions/map/map-switcher/map-switcher.component.scss

```diff
@@ -0,0 +1,24 @@
+.switcher {
```

Comment:
Replace with `:host` if we want to keep some styles.

However, instead of using absolute positioning, if we correct map parent styling as shown in `map.component.scss` related coment, this can be solved with some padding on parent map component instead.

The background and padding would be probably solved by `mat-card` as proposed in the template.

Reply:
Yeah, although absolute positioning seems to be necessary to keep this element over the map.